com.bjareholt.johan.SimpleDiary.yaml

```diff
@@ -0,0 +1,27 @@
+app-id: com.bjareholt.johan.SimpleDiary
+runtime: org.gnome.Platform
+runtime-version: '3.38'
+sdk: org.gnome.Sdk
+command: simple-diary
+modules:
+  - name: md4c
+    buildsystem: cmake
+    sources:
+      - type: git
+        url: https://github.com/mity/md4c.git
+        tag: release-0.4.7
+  - name: simple-diary
+    buildsystem: meson
+    sources:
+      - type: git
+        url: https://github.com/johan-bjareholt/simple-diary-gtk.git
+        tag: v0.1.3
+finish-args:
+  # Wayland access
+  - --socket=wayland
+  # fallback to X11 + XShm access
+  - --socket=fallback-x11
+  - --share=ipc
+  - --device=dri
+  # Needs to save diary locally and it's user-configurable
+  - --filesystem=host
```

Comment:
Ideally it should either use GtkFileChooserNative to delegate permission handling to portal or use `$XDG_DATA_HOME` to store data (and where your application has write access by default).

Reply:
> Ideally it should either use GtkFileChooserNative to delegate permission handling to portal

If I understand GtkFileChooserNative from the GTK documentation correctly that's not possible. The application creates a folder where it saves multiple files and it would be quite cumbersome if the user had to choose that folder on every start-up.

> or use $XDG_DATA_HOME to store data (and where your application has write access by default).

An acceptable compromise, but that limits the user to be able to save their entries on a seperate drive than the home folder relies (unless it's symlinked?). Fixed